cmd/webhook/main.go

```diff
@@ -16,19 +16,27 @@ package main
 
 import (
 	"crypto/tls"
+	"context"
 	"flag"
 	"fmt"
 	"net/http"
+	"os"
 	"time"
 
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
 	"github.com/fsnotify/fsnotify"
 	"github.com/golang/glog"
 	"github.com/intel/network-resources-injector/pkg/webhook"
 )
 
-const defaultClientCa = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+const (
+	defaultClientCa              = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+	customizedInjectionConfigMap = "nri-customized-injections"
```

Comment:
would this be only one global config map for all the pods  ?  should we update same existing config map if there is a need to add/update/delete data field ?

Reply:
Yes, this will be the only global configmap for all pods. A pod can decide which data field shall be applied by using different labels.   This single configmap shall be updated when customized injection will be added/updated/deleted.